lib/OrdinaryDiffEqLowStorageRK/src/algorithms.jl

```diff
@@ -890,7 +890,7 @@ end
 
 #SSP Optimized Runge-Kutta Methods
 
-@doc explicit_rk_docstring("TBD",
+@doc explicit_rk_docstring("To be done",
```

Comment:
@ChrisRackauckas, write another sentence, please.

Reply:
"Optimal Strong-Stability-Preserving Runge–Kutta Time Discretizations for Discontinuous Galerkin Methods" by Kubatko, Yeager, Ketcheson (2014)

Reply:
So why is this solver in `OrdinaryDiffEqLowStorageRK `and not in `OrdinaryDiffEqSSPRK`, since I'm guessing SSPRK stands for strong stability preserving runge kutta.

Reply:
It is both SSP and a low storage method IIUC? @ranocha

Reply:
Yes, but I think it should be in OrdinaryDiffEqSSPRK.jl, see https://github.com/SciML/OrdinaryDiffEq.jl/issues/2373
Many SSP methods are low-storage schemes, but we still want to collect them in the SSP sub-package